nix/build/hacks.nix

```diff
@@ -27,6 +35,9 @@ with oself; {
   conf-libev = self.libev;
   conf-hidapi = self.hidapi;
   conf-pkg-config = self.pkg-config;
+  conf-rust = null;
```

Comment:
:joy:

Reply:
Oh right, yeah. Well. Rust doesn't build with Musl, so it will have to be done in a super-hacky way by reimporting nixpkgs.

Reply:
Btw, I've recently tried to build new tezos with rust deps with musl inside the alpine image and everything went just fine

Reply:
```
$ p pkgsMusl.rustc
error: --- ThrownError --- nix-shell
missing bootstrap url for platform x86_64-unknown-linux-musl
(use '--show-trace' to show detailed location information)

```

Reply:
I guess we can use `pkgsCross.musl64` but ocaml doesn't like that